runtime/doc/provider.txt

```diff
@@ -58,12 +58,14 @@ If you run into problems, uninstall _both_ then install "pynvim" again: >
 PYTHON PROVIDER CONFIGURATION ~
 						*g:python_host_prog*
 Command to start Python 2 (executable, not directory). Setting this makes
-startup faster. Useful for working with virtualenvs.  >
+startup faster. Useful for working with virtualenvs. Must be set before any
+check for |has|("python2").  >
```

Comment:
The `|has|` link is not a valid tag so just remove the bars. Same for other case.

Reply:
Done; thanks. (I’m not super familiar with this help file syntax; I’d
assumed that it’d work because `:help has` resolves unambiguously, but
will happily defer to your expertise here.)